models/experimental/functional_unet/tt/model_preprocessing.py

```diff
@@ -46,11 +46,11 @@ def create_unet_model_parameters(
     for key in parameters.keys():
         parameters[key].module = getattr(model, key)
 
-    parameters.c1["conv_blocking_and_parallelization_config_override"] = {"act_block_h": 16 * 32}
+    parameters.c1["conv_blocking_and_parallelization_config_override"] = {"act_block_h": 8 * 32}
```

Comment:
Is this a device performance regression?

Reply:
There is some strange pcc error issue faced in unet by pavle and me. I asked pavle and he had this workaround. I have mentioned same in latest commit(Suggested by field).
I have taken it from pavle's branch `https://github.com/tenstorrent/tt-metal/tree/pjosipovic/fix_cb_asserts`